### ISSUE TITLE
CLI driver missing for s390x arch 	s390x64_odbc_cli.tar.gz causing failures in pip install ibm-db

### ISSUE THREAD

**Issue by manogy** [ibmdb/python-ibmdb#760]
- Operating System Name: Linux
- db2level output from clidriver if in use:  NA
- Target Db2 Server Version:  NA
- Python Version: 3.9
- ibm_db version:  all versions 
- For non-Windows, output of below commands:
  name : Linux
  uname -m. : s390x
- Value of below environment variables if set:
  IBM_DB_HOME:  NA
  PATH:  NA
  LIB/LD_LIBRARY_PATH/DYLD_LIBRARY_PATH:  NA

- Test script to reproduce the problem.
- For installation related issue, complete output of `pip install ibm_db` command.
 ```
 (manogya) [root@manogya1 vision-release-0.11]# pip install ibm-db
Collecting ibm-db
  Using cached ibm_db-3.1.2.tar.gz (1.1 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
ERROR: Could not install packages due to an OSError: [Errno 2] No such file or directory: '/tmp/tmp53ju7509/output.json'

(manogya) [root@manogya1 vision-release-0.11]# 
```
Steps to Reproduce:

1.  run pip install ibm-db

**Comment by venkataramanam** [comment]
We happened to contact the concerned maintainer of the file server - https://public.dhe.ibm.com/ibmdl/export/pub/software/data/db2/drivers/odbc_cli/
This should have been resolved now.

**Comment by bimalkjha** [comment]
@cemmanuelsr Yes, ibm_db==3.1.2 has issue but that is not related to the missing clidriver. clidriver is available now. We'd suggest to install version 3.1.1 if installation of 3.1.2 fails.
There is some bug in the code of version 3.1.2 which need to be fixed. I would suggest to paste the details about error. It is something around parsing of output of db2level command. Thanks.
